oop/src/views/ReviewPage.vue

```diff
@@ -82,6 +82,13 @@ export default {
 
           switch (returnCode) {
             case '0': // Submission successful.
+              // Append city and state into one field - addressLines[1] if country is USA
+              if(this.$store.state.form.country == 'United States'){
+                const appendedCityState = this.$store.state.form.addressLines[1].value + ' ' + this.$store.state.form.province + ' USA';
+                this.$store.state.form.addressLines[1].value = appendedCityState;
+                this.$store.state.form.province = null;
```

Comment:
With Vuex, the idea is to never manually update the state. The correct way to update a state property is to dispatch an action, which triggers a mutation, which then makes the assignment. So you should be dispatching an action here instead.